src/ui/components/Toolbar.tsx

```diff
@@ -210,6 +207,15 @@ export default function Toolbar() {
           />
         ) : null}
 
+        {replayAssistExperimentEnabled && showReplayAssist ? (
```

Comment:
This place should be checking both the Redux state (if we decide to keep this) _and_ the experimental setting (so that it doesn't leak).

Personally I think the redundant Redux state is a bit odd.